src/debug.ts

```diff
@@ -0,0 +1,18 @@
+export const debug = {
```

Comment:
This seems to work, right?

Reply:
You could also just export each funtion separately, e.g. `export function log (...args: any[])` instead of packing it up in an object.

Reply:
Done in c5f47f0